gpc-api-facade/src/test/java/uk/nhs/adaptors/pss/gpc/util/fhir/OperationOutcomeUtilsTest.java

```diff
@@ -15,16 +14,17 @@ public class OperationOutcomeUtilsTest {
 
     @Test
     public void testCreateOperationOutcome() {
+
         var details = createCodeableConcept();
         var diagnostics = "Dancing not supported";
 
-        OperationOutcome result = OperationOutcomeUtils.createOperationOutcome(NOTSUPPORTED, ERROR, details, diagnostics);
+        var result = OperationOutcomeUtils.createOperationOutcome(NOTSUPPORTED, ERROR, details, diagnostics);
 
         assertThat(result.getMeta().getProfile().get(0).equals(URI_TYPE)).isTrue();
-        assertThat(result.getIssueFirstRep().getCode()).isEqualTo(NOTSUPPORTED);
-        assertThat(result.getIssueFirstRep().getSeverity()).isEqualTo(ERROR);
-        assertThat(result.getIssueFirstRep().getDetails()).isEqualTo(details);
-        assertThat(result.getIssueFirstRep().getDiagnostics()).isEqualTo(diagnostics);
+        assertEquals(NOTSUPPORTED, result.getIssueFirstRep().getCode());
+        assertEquals(ERROR, result.getIssueFirstRep().getSeverity());
+        assertEquals(details, result.getIssueFirstRep().getDetails());
+        assertEquals(diagnostics, result.getIssueFirstRep().getDiagnostics());
```

Comment:
While I love a good refactor, mixing functionality with refactors within the same PR can be a bit unclear when looking at the change whether something has intentionally changed or not.

Reply:
I just thought that this refactoring is quite simple and straightforward (all changes in one place) hence decided to include it in here